app/(main)/page.tsx

```diff
@@ -41,6 +60,7 @@ export default async function Home() {
             <InterviewCard key={interview.id} interview={interview} />
           ))}
         </div>
+        <DataTable columns={InterviewColumns} data={interviews} />
```

Comment:
I know this implementation is really just an example, but I think the ideal way of implementing this would be:

- Add the `<DataTable />` to the root `components` folder.
- Create a `<InterviewsTable />` component in the _route folder_ (assuming we anticipate it will only be used on a single route. So in this case, putting it in `app/(main)/_components`. 
- Ideally, putting the component in its own subfolder too, which would allow the interview specific column definitions in `Columns.tsx` to be moved there, too.
- Not sure if this is possible, but providing a _default_ columns definition that will be automatically generated based on whatever the structure of the data parameter is. So obviously nothing fancy like sorting or selecting - just rendering a table.

Reply:
And then doing the same for `<ProtocolsTable />`, `<ParticipantsTable />` etc...

Reply:
Resolved with [480c1c1](https://github.com/complexdatacollective/Fresco/pull/6/commits/480c1c11df0dcead08587a36bd746ddbc3f73c7c), [4ee0915](https://github.com/complexdatacollective/Fresco/pull/6/commits/4ee091543ee1b8fa2f1faf445f0ff9e9c2b3f6ac), [22c526c](https://github.com/complexdatacollective/Fresco/pull/6/commits/22c526c1bf50733989da048bc9145da3a6ac0fe1)